common/pigx-runner.in

```diff
@@ -563,4 +563,9 @@ else:
         command = command + args.snakemake
     if args.target and 'help' in args.target:
         command.append("help")
-    subprocess.run(command)
+    command_exit = subprocess.run(command)
+
+    if command_exit.returncode == 0:
+        sys.exit(0)
+    else:
+        sys.exit(1)
```

Comment:
why not `sys.exit(command_exit.returncode)`?  There are more return codes than 0 and 1, so we could just pass the raw value on.

Reply:
I considered that, but i thought the pipeline might need its own exit codes independent of snakemakes... But I suppose when that happens we can still change it back.